buildSrc/src/main/kotlin/Publishing.kt

```diff
@@ -0,0 +1,55 @@
+import org.gradle.api.Project
+import org.gradle.api.publish.PublishingExtension
+import org.gradle.api.publish.maven.MavenPublication
+import org.gradle.api.tasks.SourceSetContainer
+import org.gradle.jvm.tasks.Jar
+import org.gradle.kotlin.dsl.create
+import org.gradle.kotlin.dsl.get
+
+internal const val PUBLICATION_NAME = "koptional-publication"
+
+fun PublishingExtension.configure(project: Project) {
+    publications {
+        create<MavenPublication>(PUBLICATION_NAME) {
+            from(project.components["java"])
+
+            groupId = "com.gojuno.koptional"
+            artifactId = project.name
+            version = project.gitVersion()
+
+            artifact(createSourcesTask(project))
+
+            pom {
+                name.set(project.name)
+                description.set(project.description)
+                url.set("https://github.com/gojuno/koptional")
+
+                developers {
+                    developer {
+                        id.set("gojuno")
+                        name.set("Juno Inc.")
+                        email.set("opensource@gojuno.com")
```

Comment:
By the way, are gojuno emails still alive?

Reply:
This is a good point. Everything Juno-related is pretty much dead. I’m trying to transfer the Bintray account but it’s complicated by the fact that the company doesn’t exist, emails don’t exist and so on. Hopefully we can resolve this, otherwise we are pretty much locked out of managing the artifact. The only thing that works is secrets on Travis.